sunpy/map/tests/test_mapbase.py

```diff
@@ -635,9 +658,9 @@ def test_as_mpl_axes_aia171(aia171_test_map):
 
 def test_pixel_to_world_no_projection(generic_map):
     with pytest.warns(SunpyUserWarning, match='Missing metadata for observer'):
-        out = generic_map.pixel_to_world(*u.Quantity(generic_map.reference_pixel)+1*u.pix, origin=1)
-    assert_quantity_allclose(out.Tx, -10*u.arcsec)
-    assert_quantity_allclose(out.Ty, 10*u.arcsec)
+        out = generic_map.pixel_to_world(*u.Quantity(generic_map.reference_pixel))
+    assert_quantity_allclose(out.Tx, 0*u.arcsec)
+    assert_quantity_allclose(out.Ty, 0*u.arcsec)
```

Comment:
Now the coordinate of the reference pixel is the same as the coordinate in the WCS header 🎉

Reply:
well the issue was we were doing +1 and `origin=1` surely just `origin=1` would have been sufficient :facepalm: